ghost/email-service/i18n-setup.js

```diff
@@ -0,0 +1,15 @@
+const i18nLib = require('../i18n');
```

Comment:
I _think_ you don't want to import files across package boundaries - you would need to add the package as a dependency and import from it